consensus/ibft/ibft.go

```diff
@@ -679,12 +684,13 @@ func (i *Ibft) writeTransactions(gasLimit uint64, transition transitionInterface
 		// no errors, pop the tx from the pool
 		i.txpool.Pop(tx)
 
-		successful = append(successful, tx)
+		successTxCount++
+		transactions = append(transactions, tx)
 	}
 
-	i.logger.Info("picked out txns from pool", "num", len(successful), "remaining", i.txpool.Length())
+	i.logger.Info("picked out txns from pool", "numFailedTx=", failedTxCount, "numSuccessTx=", successTxCount, "remaining", i.txpool.Length())
```

Comment:
You don't need to add `=`

Reply:
fixed 8b76af4b0bc360d8713ec51fd4df7e779114d994